integration/flink/shared/src/main/java/io/openlineage/flink/client/Versions.java

```diff
@@ -0,0 +1,34 @@
+/*
+/* Copyright 2018-2024 contributors to the OpenLineage project
+/* SPDX-License-Identifier: Apache-2.0
+*/
+
+package io.openlineage.flink.client;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URI;
+import java.util.Properties;
+
+public class Versions {
+
+  public static final URI OPEN_LINEAGE_PRODUCER_URI = getProducerUri();
+
+  private static URI getProducerUri() {
+    return URI.create(
+        String.format(
+            "https://github.com/OpenLineage/OpenLineage/tree/%s/integration/flink", getVersion()));
+  }
+
+  @SuppressWarnings("PMD")
```

Comment:
Please mention particular warning you want to suppress.

Reply:
@pawel-big-lebowski to be honest, I've copied this fragment from Spark integration. Should I change this there as well?